plugin-template

```diff
@@ -328,26 +327,25 @@ def main():
     except Exception:
         config["current_version"] = "0.1.0a1.dev"
 
+    # Determine if plugin is a member of Pulp managed documentation
+    config["is_pulpdocs_member"] = config["plugin_name"] in utils.get_pulpdocs_members()
```

Comment:
I like it. When onboarding a new plugin to the docs, we need to be extra careful. (Just a reminder at this point.)